.github/workflows/e2e-init.yaml

```diff
@@ -165,6 +166,7 @@ jobs:
           make ${{ matrix.e2e_test_mode }} -e E2E_CLUSTER_NAME=${{ env.E2E_CLUSTER_NAME }}  \
               -e E2E_GINKGO_LABELS=${E2E_LABELS} \
               -e E2E_TIMEOUT=${{ env.E2E_TIME_OUT }} \
+              -e INSTALL_OVS=${{ env.INSTALL_OVS }} \
```

Comment:
use the E2E codes to check whether we have ovs SMC or IPPool resource

Reply:
I don't need ovs, I just need to confirm that the node has an additional network card for network card abnormality testing, such as eth1.

But you're right, I shouldn't judge in CI, put it in It.